docker_run.sh

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Can you add some examples to the readme on how to run using this script ?

Reply:
Does this actually work with libvirt ? I.e. no need to mount the images into it ? or I'm missing something here ?

Reply:
It's not using the libvirt, but try to run avocado in docker (centos8)

```
Build docker image:
  docker build -f ./Dockerfile . -t amoskong/artifact-test-centos8

Run artifact-test:
  ./docker_run.sh amoskong/artifact-test-centos8 avocado run scylla-artifacts.py -m scylla-artifacts.yaml --job-results-dir .
```

I will add a README.txt